src/test/scala/com/bwsw/cloudstack/vault/server/kafka/ConsumerTestSuite.scala

```diff
@@ -108,7 +108,7 @@ class ConsumerTestSuite extends FlatSpec with Matchers {
     assert(consumer.process().isInstanceOf[Unit])
   }
 
-  "process" should "does not swallowed non-CriticalException" in {
+  "process" should "throw AbortedException if event processing throw non-CriticalException" in {
```

Comment:
"process" should "throw AbortedException if non-CriticalException has been thrown during event processing" in

Reply:
ok